jekyll/_cci2/testing-llm-enabled-applications-through-evaluations.adoc

```diff
@@ -26,7 +21,7 @@ Evaluations provide insights into the performance and efficiency of applications
 
 The term _evaluation_ initially referred to a way to rate and compare AI models. It has since expanded to include application-level testing, including Retrieval Augmented Generation (RAG), function calling, and agent-based applications.
```

Comment:
```suggestion
The term _evaluation_ initially referred to a way to rate and compare AI models. Evaluation has since expanded to include application-level testing, including Retrieval Augmented Generation (RAG), function calling, and agent-based applications.
```